Source/Scene/JobScheduler.js

```diff
@@ -3,6 +3,11 @@ import DeveloperError from "../Core/DeveloperError.js";
 import getTimestamp from "../Core/getTimestamp.js";
 import JobType from "./JobType.js";
 
+/**
+ *
+ * @private
+ * @constructor
```

Comment:
I don't think this `@constructor` was intentional.

Reply:
It was added there so that JSDoc doesn't mark it as a global member. @lilleyse and I found that adding `@constructor` was needed because the `@private` isn't inherited